src/js/cart.js

```diff
@@ -25,4 +25,11 @@ function cartItemTemplate(item) {
   return newItem;
 }
 
+function renderCartTotal() {
+  const cart = getLocalStorage('so-cart') || [];
+  const total = cart.reduce((sum, item) => (sum += item.FinalPrice), 0);
```

Comment:
Step 2: Reduce the items in the cart to a sum total 

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

A loop with a `let sum = 0;` works too.